eclair-core/src/main/resources/reference.conf

```diff
@@ -347,6 +347,10 @@ eclair {
       "mempool.space"
     ]
   }
+
+  onion-messages {
+    max-per-peer-per-second = 10
```

Comment:
I think it will be worth rebasing this PR on top of #2089 and create a new `OnionMessagesConfig` object in `NodeParams` that will reflect all the fields we'll put in here (more future-proof than adding individual fields to `NodeParams` every time).